sql/bodfeedback.sql

```diff
@@ -0,0 +1,32 @@
+DROP TABLE IF EXISTS bod_complaint_emails;
+DROP TABLE IF EXISTS bod_complaint_messages;
+DROP TABLE IF EXISTS bod_complaint_info;
+
+CREATE TABLE bod_complaint_info (
+  complaint_id int(11) NOT NULL AUTO_INCREMENT,
+  subject varchar(50) DEFAULT NULL,
+  status varchar(36) DEFAULT NULL,
+  uuid varchar(36) NOT NULL,
```

Comment:
Use `BINARY(16)` with `UUID_TO_BIN()` and `BIN_TO_UUID()` as described here: www.mysqltutorial.org/mysql-uuid/

Reply:
UUID_TO_BIN() is only in MySQL not in MariaDB I think.
https://mariadb.com/kb/en/library/function-differences-between-mariadb-103-and-mysql-80/

Reply:
Use `UNHEX(REPLACE(UUID(), '-', ''))` to generate a UUID and `HEX(uuid)` to convert it back to hexadecimal